Assets/StreamingAssets/Text/Internal_Locations.csv

```diff
@@ -0,0 +1,15252 @@
+﻿Key,Value
```

Comment:
That file doesn't exist under this path on master branch, it can only be found under Assets/StreamingAssets/Text/Master Localization CSV Files